src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Interfaces/IProcessDumpUtility.cs

```diff
@@ -28,7 +28,24 @@ public interface IProcessDumpUtility
         /// <param name="isFullDump">
         /// Is full dump enabled
         /// </param>
-        void StartProcessDump(int processId, string dumpFileGuid, string testResultsDirectory, bool isFullDump = false);
+        void StartTriggerBasedProcessDump(int processId, string dumpFileGuid, string testResultsDirectory, bool isFullDump = false);
```

Comment:
>string testResultsDirectory [](start = 78, length = 27)

suggestion: string testResultsDirectory, processId could be part of an Initialize method .. that way you can scope an instance of this class for a specific process and not leave it wide open.

Reply:
Yeah, existing UTs are what dissuaded me from making this change. Wanted to avoid refactoring interfaces as much as possible given that i had limited time